web/src/loading.ts

```diff
@@ -92,3 +92,27 @@ export function show_button_spinner($elt: JQuery, using_dark_theme: boolean): vo
     }
     $elt.css("display", "inline-block");
 }
+
+// For showing spinners at dialog_widget and user_profile.
+export function show_spinner($button: JQuery): void {
```

Comment:
Let's rename `show_spinner` to `show_dialog_spinner` or `show_modal_spinner`.

Reply:
`show_spinner` sounds like a very general use case, but the logic of this function atm works only for dialogs/modals.